check_api/src/main/java/com/google/errorprone/fixes/SuggestedFixes.java

```diff
@@ -734,7 +734,7 @@ public CharSequence getCharContent(boolean ignoreEncodingErrors) throws IOExcept
     try {
       newTask.analyze();
     } catch (Throwable e) {
-      e.printStackTrace();
+      // ignored
```

Comment:
@cushon, have you considered doing `return false` instead, as suggested in #849? Cause as-is this change only makes the issue in #849 harder to debug.